controller/app/helpers/admin_helper.rb

```diff
@@ -99,8 +99,14 @@ def get_user_info(user)
     user["ssh_keys"].each { |k| user_ssh_keys << ["#{user['_id'].to_s}-#{k['name']}", Digest::MD5.hexdigest(k["content"]), nil] if k["content"] } if user["ssh_keys"].present?
     user_caps = user["capabilities"]
     if user["plan_id"]
-      plan_caps = OpenShift::BillingService.instance.get_plans[user["plan_id"].to_sym][:capabilities]
-      user_caps = plan_caps.deep_dup.merge(user_caps)
+      plans = OpenShift::BillingService.instance.get_plans[user["plan_id"].to_sym]
```

Comment:
I can definitely change plans to plan to make the variable more accurate.  I'm pretty sure that this is correct though for the call to get_plans().  The user["plan_id"] is going to come back with a string such as "free".  It converts that to a symbol :free, and then runs checks get_plans[:free] and sets that to plan, which we're looking at to see if it is nil.